--- a/CMakeLists.txt	2019-01-30 08:37:10.000000000 -0600
+++ b/CMakeLists.txt	2019-02-28 15:49:07.218026681 -0600
@@ -17,6 +17,12 @@
 option(INSTALL_MINGW_DEPENDENCIES "Add dependent libraries to MinGW install target" OFF)
 endif()
 
+# Version
+SET(LIB_MAJOR_VERSION "0")
+SET(LIB_MINOR_VERSION "19")
+SET(LIB_REVISION "02")
+SET(LIB_VERSION "${LIB_MAJOR_VERSION}.${LIB_MINOR_VERSION}.${LIB_REVISION}")
+
 # Build Type
 if(NOT CMAKE_BUILD_TYPE AND NOT CMAKE_CONFIGURATION_TYPES)
 	# By default, we use Release
@@ -103,6 +109,12 @@
 	target_link_libraries(FAudio PRIVATE -static-libgcc)
 endif()
 
+# Soname
+set_target_properties(FAudio PROPERTIES OUTPUT_NAME "FAudio"
+       VERSION ${LIB_VERSION}
+       SOVERSION ${LIB_MAJOR_VERSION}
+)
+
 # XNA_Song Support
 if(XNASONG)
 	target_sources(FAudio PRIVATE src/XNA_Song.c)
